### PR TITLE
Extension sample

### PR DESCRIPTION
**Pull request by akihikokuroda**
This is simple hello world extension example.

### CODE REVIEW

File: docs/samples/extension/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+This sample shows how to extend the tekton dashboard using the dashboard extension.  The extensions should be in experimental repository (https://github.com/tektoncd/experimental)
```
Comment (mnuttall) [inline docs/samples/extension/README.md]: > This sample shows how to extend the tekton dashboard using the dashboard extension.

"This sample shows how to extend the tekton dashboard using the extension mechanism" might be better. 

> The extensions should be in experimental repository (https://github.com/tektoncd/experimental)

Perhaps we could say, 'This sample extension is intended just to illustrate the mechanism. See the 'experimental' repository (https://github.com/tektoncd/experimental) for more extensions.'

File: docs/samples/extension/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Build sample extension image 
+
+```
+cd docs/samples/extension
+docker build -t sampleextension .
+```
+## Push the sample container image to the registory (like dockerhub)
```
Comment (mnuttall) [inline docs/samples/extension/README.md]: s/registory/registry

File: docs/samples/extension/server.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const PORT = 3000;
+const HOST = '0.0.0.0';
+
+// App
+const app = express();
+app.get('/', (req, res) => {
+	res.send('Hello tekton dashboard!!\n');
+    });
+app.get('/extensions/sample', (req, res) => {
+	res.send('Hello tekton dashboard!! Here is /extenstions/sample\n');
```
Comment (mnuttall) [inline docs/samples/extension/server.js]: /extensions/sample

File: docs/samples/extension/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ }
+]
+```
+## Access to the extension
+```
+http://localhost:9097/extensions/sample
+```
+### Output:
+```
+Hello tekton dashboard!! Here is /extenstions/sample
```
Comment (mnuttall) [inline docs/samples/extension/README.md]: I've added a comment below - this should read '/extensions/sample' once the code is fixed

File: docs/samples/extension/config/tekton-extension-sample-service.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: dashboard-extension
+  labels:
+    tekton-dashboard-extension: "true"
+  annotations:
+    tekton-dashboard-endpoints: "/extensions/sample"
+    tekton-dashboard-display-name: "tekton_dashboard_extension"
+    tekton-dashboard-bundle-location: "/bundle"
```
Comment (mnuttall) [inline docs/samples/extension/config/tekton-extension-sample-service.yaml]: I won't be able to test this until Monday at the earliest, but https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/ states, 
> The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between. 

Are the / characters valid and accepted - do they work?

Comment (akihikokuroda) [reply docs/samples/extension/config/tekton-extension-sample-service.yaml]: Yes, it works.

File: docs/samples/extension/README.md (hunk 4 above)
Comment (akihikokuroda) [reply docs/samples/extension/README.md]: I misunderstood.  Yes, I fix it.

**Comment by akihikokuroda** [comment]
The sample is updated and retested with the latest extension PR.

**Pull request merged**